### PR TITLE
Add "Show name" option to Telegram presence

### PR DESCRIPTION
**Pull request by playday3008**
#2700

### CODE REVIEW

File: websites/T/Telegram/presence.ts [hunk 1]
```diff
@@ -5,6 +5,8 @@ var presence = new Presence({
 presence.on("UpdateData", async () => {
   const path = document.location.href;
 
+  const showName = await presence.getSetting("name");
+
   const presenceData: PresenceData = {
```
Comment (ririxidev) [inline websites/T/Telegram/presence.ts:10]: merge consts thanks

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
and uk isnt valid lang xd

**Comment by playday3008** [comment]
> and uk isnt valid lang xd

tested with uk_UA, doesn't work in Microsoft Edge

**Comment by playday3008** [comment]
for example, SoundCloud
image
image

File: websites/T/Telegram/presence.ts (hunk 1 above)
Comment (playday3008) [reply websites/T/Telegram/presence.ts:10]: Explain please

**Comment by ririxidev** [comment]
not our issue that edge handles it differently ://///

File: websites/T/Telegram/presence.ts (hunk 1 above)
Comment (ririxidev) [reply websites/T/Telegram/presence.ts:10]: bruh
```javascript
const f = "fgf",
g = "sgsg",
h = "afaf";
```

Comment (playday3008) [reply websites/T/Telegram/presence.ts:10]: const path = document.location.href,
showName = await presence.getSetting("name");
?

Comment (ririxidev) [reply websites/T/Telegram/presence.ts:10]: you have three consts to merge

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of it working

**Comment by playday3008** [comment]
> Upload proof of it working

image
image

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**